gatsby-remark-oembed/utils/getProviderEndpointForLinkUrl.js

```diff
@@ -7,6 +7,14 @@ const getProviderEndpointForLinkUrl = (linkUrl, providers) => {
         schema = schema.replace("*", ".*");
         const regExp = new RegExp(schema);
         if (regExp.test(linkUrl)) {
+          if (provider.provider_name === "Instagram") {
```

Comment:
refactor (non-blocking): These do not need to be two `if` blocks. It can be one with a consolidated condition.

Reply:
Did the first two, will break the third out as an issue.

Reply:
> refactor (blocking): Move the error check into the providers loop so that we can throw the error as soon as we're on the Instagram provider.

Just realized this will not give us the intended behaviour. We need to check if a oembed link is in fact an Instagram link by checking against the schemas. If there are no instagram links, no need to let anyone know there an access token needs to be configured. Will make a test case for this, so we remember in the future.